src/main.cpp

```diff
@@ -64,8 +64,49 @@ static void servoSetUs(unsigned int servo, int usec)
     g_OutputsUs[servo] = usec;
 }
 
+
+static void outputFailsafeValues()
+ {
+    for (unsigned int out=0; out<NUM_OUTPUTS; ++out)
+        {
+            if (OUTPUT_FAILSAFE[out] == fsaNoPulses)
+                servoSetUs(out, 0);
+            else if (OUTPUT_FAILSAFE[out] != fsaHold)
+                servoSetUs(out, OUTPUT_FAILSAFE[out]);
+            // else fsaHold does nothing, keep the same value
+        }
+}
+
+
+#if defined(ARMSWITCH)
+    short armCount = 0;
+#endif

```

Comment:
global variable in the middle of nowhere? Move to static inside handler, `uint8_t` type